lib/command/src/Obelisk/Command.hs

```diff
@@ -161,6 +161,9 @@ deployInitOpts = DeployInitOpts
   <$> strArgument (action "directory" <> metavar "DEPLOYDIR" <> help "Path to a directory that it will create")
   <*> strOption (long "ssh-key" <> action "file" <> metavar "SSHKEY" <> help "Path to an ssh key that it will symlink to")
   <*> some (strOption (long "hostname" <> metavar "HOSTNAME" <> help "hostname of the deployment target"))
+  <*> strOption (long "route" <> metavar "PUBLICROUTE" <> help "Public URL to your app (comon/route)")
```

Comment:
Let's take out `(common/route)` here.  Without further explanation, that won't mean anything to people.